alacritty_terminal/src/renderer/mod.rs

```diff
@@ -297,6 +368,13 @@ impl GlyphCache {
             let mut rasterized =
                 rasterizer.get_glyph(glyph_key).unwrap_or_else(|_| Default::default());
 
+            // FIXME
+            // Assume that all emojis are 2 width for now.
+            let h = metrics.line_height as i32;
```

Comment:
We can pass the WIDECHAR flag and just multiply it properly, I guess.

Reply:
I mean don't use height for both things, this was just for showing what we need and where.